Makefile

```diff
@@ -155,6 +157,14 @@ oc-commands: commands ## Generate oc plugins and add them to build folder
 install-commands: commands ## Generate plugins and add them to /usr/bin/
 	sudo cp -a ./build/. /usr/bin/
 
+.PHONY: release
+release: clean ## Generate tar.gz containing krew plugin and display krew updated index
+	$(MAKE) KREW_PLUGIN=true kubectl-commands
+	tar -czf netobserv-cli.tar.gz LICENSE ./build/netobserv
+	@echo "### Generating krew index yaml"
+	VERSION=$(VERSION) \
+	./scripts/krew.sh
```

Comment:
don't we need this operation to be arch aware and generate tar ball for each arch ?

Reply:
I need to test it on different arch first. It may work out of box; else maybe just tweaking around the script.

On top, krew index allow you to point different binaries for each platform if needed